pkg/appgateserver/server.go

```diff
@@ -159,26 +170,43 @@ func (app *appGateServer) ServeHTTP(writer http.ResponseWriter, request *http.Re
 			ctx,
 			requestPayloadBz,
 			writer,
+			serviceId,
+			"unknown",
 			ErrAppGateHandleRelay.Wrapf("reading relay request body: %s", err),
 		)
 		// TODO_TECHDEBT: log additional info?
 		app.logger.Error().Err(err).Msg("failed reading relay request body")
+
 		return
 	}
 	app.logger.Debug().
 		Str("service_id", serviceId).
 		Str("payload", string(requestPayloadBz)).
 		Msg("handling relay")
 
+	// TODO_TECHDEBT: log additional info?
```

Comment:
```suggestion
	// TODO_IMPROVE: log additional info?
```

In order of importance:
TODO_BLOCKER
TODO_TECHDEBT
TODO_IMPROVE